### DISCUSSION TITLE
Mac x86-64 version not recongnize dosbox-x.conf in current dir.

### DISCUSSION THREAD

**Discussion by yochenhsieh** [joncampbell123/dosbox-x#4726] [Q&A]
OS: Mac OS X 12.7.2
DosBox-X Version: 2023.05.01 (cannot run 202309 and 202310 release)
For dosbox, if I put dosbox.conf and dosbox.app in same directory, dosbox will use that config automatically. 
For dosbox-x, it seems cannot recognize dosbox-x.conf and still request to set a working directory at start. If I select cancel, it asks if I want to use current path, I select yes. But then it still won't use the config and starts with Z:\>

Is this by design? Why does it behave differently? 
Or did I hit a bug?

**Comment by joncampbell123** [comment]
I suppose that counts as a bug.

**Reply by yochenhsieh** [reply]
OK, reported. Thank you!

**Reply by Torinde** [reply]
Since you opened #4727, please close this discussion.

**Comment by yochenhsieh** [comment]
Duplicate of #4727. closed.
